### PR TITLE
Fix missing shuffling functionality in masking command

### PR DESCRIPTION
**Pull request by sanderstad**
## Type of Change

 - [X] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [X] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
Somewhere along the way the functionality for shuffling the data was lost.
The functionality is now added back in.

Another thing that was not possible was to use all the different masking types in composites.

### Approach
Added functionality in the command to shuffle text and numbers.
Added functionality to allow for masking types in composites

### Commands to test
Invoke-DbaDbDataMaskingConfig

### Screenshots
The column "Comment" has the original value "Just some comment"
image

### CODE REVIEW

File: functions/Invoke-DbaDbDataMasking.ps1 [hunk 1]
```diff
@@ -416,12 +416,16 @@ function Invoke-DbaDbDataMasking {
                                     try {
                                         $newValue = $null
 
-                                        if (-not $columnobject.SubType -and $columnobject.ColumnType -in $supportedDataTypes) {
+                                        if ($columnobject.SubType.ToLowerInvariant() -eq 'shuffle') {
+                                            $newValue = Get-DbaRandomizedValue -RandomizerType "Random" -RandomizerSubtype "SHuffle" -Value ($row.$($columnobject.Name)) -Locale $Locale
```
Comment (wsmelton) [inline functions/Invoke-DbaDbDataMasking.ps1]:
```suggestion
                                            $newValue = Get-DbaRandomizedValue -RandomizerType "Random" -RandomizerSubtype "Shuffle" -Value ($row.$($columnobject.Name)) -Locale $Locale
```

**Comment by wsmelton** [comment]
@sanderstad while you are touching this command can you correct a line for me to fix OTSB formatting?

Prior to this PR I have line 565 but not sure what line it would be in this one now, but you have:

```
                                $compositeItems = $compositeItems | ForEach-Object {
                                    $_ = "ISNULL($($_), '')"
                                    $_
                                }
```

This needs to be changed to this:
```
$compositeItems = $compositeItems | ForEach-Object { $_ = "ISNULL($($_), '')"; $_ }
```

The way Pester validates OTSB it flags this for indentation being inconsistent. If you can fix that I'll exclude it from my PR so I don't have to deal with merge conflicts.

**Comment by sanderstad** [comment]
@wsmelton Will do

**Comment by potatoqualitee** [comment]
yall good?

**Comment by sanderstad** [comment]
I think so, what do you think @wsmelton

**Comment by potatoqualitee** [comment]
awesome! thank you both

**Pull request merged**